volume/local/local_test.go

```diff
@@ -12,6 +12,22 @@ import (
 	"github.com/docker/docker/pkg/mount"
 )
 
+func TestGetAddress(t *testing.T) {
+	cases := map[string]string{
+		"addr=11.11.11.1":   "11.11.11.1",
+		" ":                 "",
+		"addr=":             "",
+		"addr=2001:db8::68": "2001:db8::68",
+	}
+	for name, success := range cases {
+		v := getAddress(name)
```

Comment:
`getAddress` is only defined for linux, but not for Windows.
Can probably move to a platform independent file.